doc/nrf/protocols/lte/index.rst

```diff
@@ -0,0 +1,214 @@
+.. _ug_lte:
+
+LTE
+###
+
+Long Term Evolution (LTE) technology enables devices to be connected to a base station without an intermediate device or gateway.
+LTE  is the fourth-generation (4G) wireless broadband standard built on existing technologies, replacing 3G.
+LTE maximizes bandwidth (BW) utilization and supports up to 300 Mbps.
+The main key features of the LTE protocol are that it is purely packet-based, provides significantly high data rates, improves latency, uses the spectrum more efficiently, supports scalable carrier bandwidths, includes robust security features, and is designed to be backward compatible.
+LTE is managed by the 3rd Generation Partnership Project (3GPP).
+They develop and maintain the cellular standards, which are structured as releases.
+
+Nordic Semiconductor offers a wide range of cellular products that that support LTE connectivity, particularly focusing on the LTE-M and NB-IoT standards.
+For more information on the products, see `Cellular IoT SiPs`_.
+
+LTE-M and NB-IoT
+****************
+
+LTE-M is a low-power wide-area network (LPWAN) communication standard, introduced in 3GPP release 13.
+It offers higher bandwidth and lower latency compared to other similar technologies.
+This makes it suited for applications such as asset tracking, health monitors, alarm panels, environmental monitoring, and smart parking.
+
+NB-IoT, or Narrowband IoT, is another LPWAN communication standard that was introduced in 3GPP release 13.
+It is designed for indoor coverage, low cost, and long battery life.
+NB-IoT operates in a small radio frequency band of 200 kHz.
+It is ideal for applications like smart metering, environmental monitoring, and smart lighting.
+
+LTE-M and NB-IoT are part of 5G.
+
+nRF91 Series devices such as nRF9161, nRF9160, and Thingy:91 support both LTE-M and NB-IoT technologies.
+
+Comparison between LTE-M and NB-IoT
+===================================
+
+The following table shows a comparison between LTE-M and NB-IoT in terms of advantages in using any of these standards:
+
++------------------+--------------------------------+--------------------------------+
+|                  | LTE-M                          | NB-IoT                         |
++==================+================================+================================+
+|                  | Higher throughput              | Longer range                   |
++                  +--------------------------------+--------------------------------+
+|                  | Lower latency                  | Better penetration             |
++                  +--------------------------------+--------------------------------+
+|   Strengths      | Roaming                        | Power efficient at lower data  |
```

Comment:
Why is roaming a strength of LTE-M but not of NB-IoT? Does this have something to do with operator offerings?

Reply:
The link points to _nRF Connect for Desktop_ documentation front page, which probably was not the intention?

Discussed with @jounikor about this. LTE-M uses the same roaming agreements as legacy LTE, so it's more probable that roaming works with LTE-M than NB-IoT. Anyway, I would like to clarify the table so, that it's not possible to get the impression that roaming does not work with NB-IoT at all. For example, "Better roaming agreements" or something like that.

Reply:
Actually, I wanted to point to the white paper : https://infocenter.nordicsemi.com/topic/nwp_044/WP/nwp_044/lte_technology.html. Sorry about that.

I will update the table as per the above suggestion.